docs/about/showcase.soy

```diff
@@ -184,6 +184,58 @@
 {/call}
 <!-- End Brainium Studios Apps -->
 
+<!-- Uber Apps -->
+{call .company}
+  {param name: 'Uber' /}
+  {param moreInfo: quoteKeysIfJs([
+      'Uber Engineering's iOS Monorepo:
+        'https://eng.uber.com/ios-monorepo/',
+    ]) /}
+  {param items}
+    {call .item}
+      {param name: 'Uber' /}
+        {param imgSrc: 'https://lh3.googleusercontent.com/iyt_f1j2d2ildbFLDlS0Qf36NJMeRVZFBetcg-pmrkdQtN9C1wUTaFhloKUcwVVfQeg=w300-rw' /}
+        {param links: [
+            'Android':
+              'https://play.google.com/store/apps/details?id=com.ubercab',
+            'iOS':
+              'https://itunes.apple.com/us/app/uber/id368677368',
+          ] /}
```

Comment:
Travis failure is because you didn't close out the `{param}` for the `.item` here (and below).